docs/advanced-features/event-driven-packaging.md

```diff
@@ -0,0 +1,181 @@
+<!-- markdownlint-disable -->
+# Event-Driven Packaging (EDP)
+
+> EDP is currently in private preview. Please contact Quilt
+directly to discuss pricing.
+
+## Overview
+
+Data can be added to your Amazon S3 buckets manually by internal
+team members and/or external collaborators, or automatically via
+instrumentation. No matter how it arrives into your buckets, it 
+is helpful, and sometimes critical, to be notified of this data 
+migration.
+
+Quilt's *Event-Driven Packaging* service (EDP) monitors S3 object events
+for a given Amazon S3 bucket prefix and groups those S3 object
+events into logical batches via custom-defined time, space and
+prefix heuristics. A `files_ready` event is generated on the receiving
+[EventBridge](https://aws.amazon.com/eventbridge/) in order to
+notify users and/or other AWS services that this logical batch of
+S3 objects are ready to be turned into Quilt packages. *This makes
+it easy to ingest data into Quilt data packages from AWS data
+migration services* such as [Storage
+Gateway](https://aws.amazon.com/storagegateway/) and [Data
+Sync](https://aws.amazon.com/datasync/).
```

Comment:
```suggestion
Quilt's *Event-Driven Packaging* service (EDP) smartly groups one or more S3 object events
into a single `files_ready` event in [AWS EventBridge](https://aws.amazon.com/eventbridge/)
so that you can easily trigger logical events like package creation that depend on batches rather
than on individual files.

Quilt EDP works well with services like [AWS Storage
Gateway](https://aws.amazon.com/storagegateway/) and [AWS Data
Sync](https://aws.amazon.com/datasync/) that write objects to S3.
```

Reply:
Applied